cqrs-codegen/impl/.gitignore

```diff
@@ -0,0 +1,2 @@
+/Cargo.lock
```

Comment:
Btw, completely forgot to ask, should we check `cqrs-codegen/impl/Cargo.lock` into repo? WASM binaries are build as a separate crate, __not__  as a member of workspace, and so `cqrs-codegen/impl/Cargo.lock` is used (only and ever) to build WASM binaries. I feel like in this case it should be checked-in.

Reply:
@tyranron

Reply:
@ffuugoo I think we can skip it for now to not bother with updates. We can keep it under VCS in future if it will expose any problems, but I'm sure it won't.